src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeIndex_Persistence.cs

```diff
@@ -129,7 +134,7 @@ public void WriteTo(ObjectWriter writer)
         }
 
         private static SyntaxTreeIndex? ReadFrom(
-            StringTable stringTable, ObjectReader reader, Checksum checksum)
+            StringTable stringTable, ObjectReader reader, Checksum? checksum)
```

Comment:
I was looking through the code to see if the usages of checksum allowed nullable, and it appears like it should.  But the constructor for SyntaxTreeIndex is not nullable enabled
http://sourceroslyn.io/#Microsoft.CodeAnalysis.Workspaces/FindSymbols/SyntaxTree/SyntaxTreeIndex.cs,28

But the property that checksum gets assigned to is in a nullable enabled file and is specifically not null
http://sourceroslyn.io/#Microsoft.CodeAnalysis.Workspaces/FindSymbols/SyntaxTree/SyntaxTreeIndex_Persistence.cs,c8398183df844544

Might be good to update the annotations there to explicitly allow null (and nullable enable the ctor)

Reply:
sure.  will  do.